dask/dataframe/io/csv.py

```diff
@@ -393,11 +390,14 @@ def text_blocks_to_pandas(
             enforce,
             kwargs,
         ),
-        label=label,
+        parts,
+        meta=head,
+        divisions=(None,) * (len(blocks) + 1),
```

Comment:
Does passing just `None` also work? I think it'd be mildly nicer if it did.

Reply:
Yes - Good catch